lib/board.rb

```diff
@@ -1,3 +1,23 @@
 class Board
+  attr_reader :cells
 
+  def initialize
+    @cells = {"A1" => Cell.new("A1"),
```

Comment:
Great job with getting these all written out! Maybe for formatting purposes, you add an empty line above line 5 where  "A1" starts  so that the beginning bracket for the hash is on its own line like you  have done for line 21.

Reply:
Good catch. I'll change that.